src/parser/test/dune

```diff
@@ -0,0 +1,7 @@
+(library
+ (name odoc_parser_test)
+ (inline_tests)
+ (optional)
```

Comment:
I suggest removing that line. With this, tests won't run with no warning if ppx_expect isn't installed.